src/iceflow/ingest/atm1b.py

```diff
@@ -368,7 +368,24 @@ def _ilatm1bv2_data(fn: Path, file_date: dt.date) -> pd.DataFrame:
     return df
 
 
-def atm1b_data(filepath: Path) -> pd.DataFrame:
+class atm1bData(commonDataColumns):
+    # Data fields unique to ATM1B data.
+    rel_time: Series[pa.dtypes.Int32]
+    xmt_sigstr: Series[pa.dtypes.Int32]
+    rcv_sigstr: Series[pa.dtypes.Int32]
+    azimuth: Series[pa.dtypes.Int32]
+    pitch: Series[pa.dtypes.Int32]
+    roll: Series[pa.dtypes.Int32]
+    gps_pdop: Series[pa.dtypes.Int32]
+    gps_time: Series[pa.dtypes.Int32]
+    passive_signal: Series[pa.dtypes.Int32]
+    passive_footprint_latitude: Series[pa.dtypes.Int32]
+    passive_footprint_longitude: Series[pa.dtypes.Int32]
+    passive_footprint_synthesized_elevation: Series[pa.dtypes.Int32]
+
+
+@pa.check_types()
```

Comment:
We're thinking of adopting pydantic's validate_arguments decorator for earthaccess, so I'll be curious what you learn about this :)